components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/api/client/ConnectionFailedException.java

```diff
@@ -0,0 +1,15 @@
+package com.gradle.enterprise.api.client;
+
+import java.net.URL;
+
+public class ConnectionFailedException extends ApiClientException {
+
+    public ConnectionFailedException(String buildScanId, URL gradleEnterpriseServer, Throwable cause) {
```

Comment:
For both constructors, I would swap the buildScanId and gradleEnterpriseServer parameters.

Reply:
✅ Done.  I did this for all of the other exceptions as well.